canonical-web-and-design/vanilla-framework
# Add search to Vanilla docs

## Done

This PR is merging the search feature branch.
Code was already reviewed in partial PRs: #2812 #2819 #2820

## QA

To QA you need a search API token, see #2812

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/
- Search the docs
